server/src/main/java/org/elasticsearch/action/support/TransportLocalClusterStateAction.java

```diff
@@ -61,4 +65,33 @@ protected final void doExecute(Task task, Request request, ActionListener<Respon
 
     protected abstract void localClusterStateOperation(Task task, Request request, ClusterState state, ActionListener<Response> listener)
         throws Exception;
+
+    // Remove the BWC support for the deprecated ?master_timeout parameter.
+    // NOTE: ensure each usage of this method has been deprecated for long enough to remove it.
+    @UpdateForV10(owner = UpdateForV10.Owner.DISTRIBUTED_COORDINATION)
+    public static void consumeDeprecatedMasterTimeoutParameter(ToXContent.Params params) {
```

Comment:
I added a separate method for the master timeout parameter to make it more explicit. I could also merge the two methods into something like `consumeDeprecatedParameters`, as I expect these two parameters to be deprecated together frequently.